src/main/java/org/zalando/nakadi/controller/ExceptionHandling.java

```diff
@@ -259,4 +268,60 @@ private String generateErrorTraceId() {
         LOG.debug(exception.getMessage());
         return Responses.create(UNPROCESSABLE_ENTITY, exception.getMessage(), request);
     }
+
+    @ExceptionHandler(InvalidLimitException.class)
```

Comment:
I moved some handlers to specific controllers, as discussed offline. Also, when appropriate, I grouped handlers together. Sometimes we log slightly different things, or reply a bit differently, between handlers that would otherwise be good candidates for grouping. In these cases I decided to go for not grouping, as to avoid subtle changes in behaviour.

One thing that comes out of this refactoring, is that we should have a hard look at our logging strategy, and make it consistent everywhere. But I think that is a matter for another pull request.